### ISSUE TITLE
autocomplete: unit test failure on Firefox 50

### ISSUE THREAD

**Issue by levgaas** [angular/material#10402]
@ThomasBurleson

Latest success build didn't use `Firefox v.50` [link](https://travis-ci.org/angular/material/jobs/199317631)
Please look at it because `Travis CI` is now using Firefox `v.50` instead of `v.49`

So builds are failed with error:
`Firefox 50.0.0 (Windows 10 0.0.0) md-input-container directive with ng-messages should select the input value on focus FAILED`

Chrome and IE passed

**Comment by ThomasBurleson** [comment]
@topherfangio - can we fix/address this?

**Comment by ThomasBurleson** [comment]
@DevVersion - can you investigate this. This is also happening with ng 1.6.2 and md-autocomplete:

http://127.0.0.1:8080/demo/autocomplete   "Floating Label" name invalid.

```console
angular-material.min.js:10 mdInput messages show animation called on invalid messages element:  U[1]0: div.ng-scope.md-input-messages-animation.ng-inactivelength: 1__proto__: Object[0]
```

**Comment by Splaktar** [comment]
The input animation warnings are tracked in https://github.com/angular/material/issues/10461. 
The unit tests are fixed and currently passing on Firefox 58.0.2.
